### PR TITLE
updated docs for skymatch and mrs_imatch

### PR DESCRIPTION
**Pull request by mcara**
This PR addresses the issue https://github.com/STScI-JWST/jwst/issues/741 for `skymatch` and also adds documentation for `mrs_imatch`.

### CODE REVIEW

**Comment by mcara** [comment]
@hbushouse Please review/merge this commit.

File: docs/jwst/mrs_imatch/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.. math::
+    P_n(k_1,k_2,...) = \sum_{d_1=0,d_2=0,...}^{D_1,D_2,...} c_{d_1,d_2,...}^n \cdot k_1^{d_1} \cdot k_2^{d_2}  \cdot \ldots .
+
+Step Arguments
+==============
+The mrs_imatch step has two optional argument:
+
+* ``bkg_degree``: An integer background polynomial degree (Default: 1)
+
+* ``subtract``: A boolean value indicating whether computed matching
```
Comment (hbushouse) [inline docs/jwst/mrs_imatch/README.rst]: A boolean value indicating whether **the** computed matching

File: docs/jwst/mrs_imatch/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  "backgrounds" should subtracted from image data (Default: `False`).
```
Comment (hbushouse) [inline docs/jwst/mrs_imatch/README.rst]: "backgrounds" should **be** subtracted from ...

File: docs/jwst/skymatch/README.rst [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+- The ``'globalmin+match'`` algorithm combines ``'match'`` and
+  ``'globalmin'`` methods. It uses ``'globalmin'``
+  algorithm to find a baseline sky value common to all input images
+  and the ``'match'`` algorithm to "equalize" sky values among images.
+
+
+Step Arguments
+--------------
+The ``skymatch`` step has the following optional argument:
```
Comment (hbushouse) [inline docs/jwst/skymatch/README.rst]: argument**_s_** (plural)

File: docs/jwst/mrs_imatch/LICENSE.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+*******
+LICENSE
+*******
+
+.. include:: ../../../LICENSE
```
Comment (nden) [inline docs/jwst/mrs_imatch/LICENSE.rst]: Have we adopted a convention of licensing files? Is there a need to put a License file in every step?
Normally I would expect to see one License file in the root directory. Then the first line of each file is the license information. I think we shuold keep these things as standard as possible.

Comment (nden) [inline docs/jwst/mrs_imatch/LICENSE.rst]: Not that I'm asking for a change here. We should in the future go over all documentation and files and make them "standard".

Comment (hbushouse) [inline docs/jwst/mrs_imatch/LICENSE.rst]: No, we haven't decided on a convention for licensing files for JWST docs. I'm open to suggestions or examples from how things are done for, say, the hstcal package.

**Comment by mcara** [comment]
@hbushouse Thanks for the comments. Can I merge this now?

**Comment by hbushouse** [comment]
👍   Merge it!

**Pull request merged**